### PR TITLE
Update to terraform 0.12.x

### PR DESCRIPTION
**Pull request by colincoleman**

### CODE REVIEW

File: examples/default/example.tf [hunk 1]
```diff
@@ -15,47 +15,47 @@ data "aws_vpc" "main" {
...
     type             = "forward"
   }
 }
 
 resource "aws_security_group_rule" "task_ingress_8000" {
-  security_group_id        = "${module.fargate.service_sg_id}"
+  security_group_id        = module.fargate.service_sg_id
   type                     = "ingress"
   protocol                 = "tcp"
   from_port                = "8000"
```
Comment (mikael-lindstrom) [inline examples/default/example.tf]: `from_port` and `to_port` can also be converted to int.

File: examples/default/example.tf [hunk 2]
```diff
@@ -15,47 +15,47 @@ data "aws_vpc" "main" {
...
+  tags = {
     environment = "test"
     terraform   = "true"
   }
 }
 
 resource "aws_lb_listener" "alb" {
-  load_balancer_arn = "${module.fargate_alb.arn}"
+  load_balancer_arn = module.fargate_alb.arn
   port              = "80"
```
Comment (mikael-lindstrom) [inline examples/default/example.tf]: I believe `port` can be converted to int.

**Comment by colincoleman** [comment]
All the port numbers in the example now updated.

**Comment by mikael-lindstrom** [comment]
I tested this yesterday but did not have the time to make a release. Thanks for fixing!

**Pull request merged**